tests/h/auth/policy_test.py

```diff
@@ -697,13 +707,15 @@ def test_identity_returns_None_with_no_token(self, pyramid_request):
     def test_identity_returns_None_for_invalid_tokens(
         self, pyramid_request, auth_token_service
     ):
+        pyramid_request.auth_token = sentinel.auth_token
         auth_token_service.validate.return_value = None
 
         assert TokenAuthenticationPolicy().identity(pyramid_request) is None
```

Comment:
These were passing for the wrong reason, as coverage was trying to tell us. I've fixed this so we get 100% now and also have upped the coverage ratchet to match recent improvements in coverage so we'll fail if we drop unexpectedly in future.